server/data/ciagInductionClient.test.ts

```diff
@@ -2,30 +2,34 @@ import nock from 'nock'
 import type { CiagInduction } from 'ciagInductionApiClient'
 import config from '../config'
 import CiagInductionClient from './ciagInductionClient'
-import { aCiagInductionWithJobInterests } from '../testsupport/ciagInductionTestDataBuilder'
+import {
+  aCiagInductionWithJobInterests,
+  aCiagInductionWithNoRecordOfAnyPreviousWorkExperience,
+} from '../testsupport/ciagInductionTestDataBuilder'
+import aValidCiagInductionListResponse from '../testsupport/ciagInductionListResponseTestDataBuilder'
 
 describe('ciagInductionClient', () => {
   const ciagInductionClient = new CiagInductionClient()
 
-  config.apis.curious.url = 'http://localhost:8200'
-  let curiousApi: nock.Scope
+  config.apis.ciagInduction.url = 'http://localhost:8200'
```

Comment:
I can't believe what a rubbish job I did of this test class originally! Clearly it was a copy and paste job from the Curious API client test class file! 🤣